style.css

```diff
@@ -109,89 +92,32 @@ select {
   grid-template-columns: auto 1fr;
 }
 
-.three-column-grid__main {
-  display: grid;
-  grid-template-columns: repeat(3, 1fr);
-}
-.two-column-grid__center-items {
-  display: grid;
-  grid-template-columns: 1fr 1fr;
-  place-items: center;
-}
-
 .three-column-grid__expand-two {
   display: grid;
   grid-template-columns: auto 1fr auto;
-  padding: 1em 2em;
 }
 
-.place-items-center {
+.six-col-grid__expand-last {
   display: grid;
-  place-items: center;
-}
-
-.align-items-center {
-  align-items: center;
+  grid-template-columns: repeat(5, 1fr) auto;
 }
 
-.place-content-center {
+.three-row-grid__expand-two {
   display: grid;
-  place-content: center;
-}
-
-/* colors */
-
-.bg-dark {
-  background-color: rgba(var(--clr-dark));
-}
-.bg-light {
-  background-color: rgba(var(--clr-light));
-}
-.bg-accent {
-  background-color: rgba(var(--clr-accent));
-}
-.bg-white {
-  background-color: rgba(var(--clr-white));
-}
-
-.text-dark {
-  color: rgba(var(--clr-dark));
-}
-.text-light {
-  color: rgba(var(--clr-light));
-}
-.text-accent {
-  color: rgba(var(--clr-accent));
-}
-.text-white {
-  color: rgba(var(--clr-white));
-}
-
-/* typography */
-
-.ff-global {
-  font-family: var(--ff-global);
-}
-
-.fs-sm {
-  font-size: var(--fs-sm);
-}
-.fs-md {
-  font-size: var(--fs-md);
-}
-.fs-lg {
-  font-size: var(--fs-lg);
```

Comment:
I like it how you weren't scared to change or remove so much code. Not needed or right - amend or get rid. No point keeping wrong or pointless code.